lib/modules/manager/flux/readme.md

```diff
@@ -36,16 +36,18 @@ If `tag` _and_ `digest` exist then `digest` takes precedence.
 
 ### Flux system manifests support
 
-Support updating Flux system manifests generated during [Flux installation](https://fluxcd.io/docs/installation/#customize-flux-manifests).
+Renovate supports updating Flux system manifests generated during [Flux installation](https://fluxcd.io/docs/installation/#customize-flux-manifests).
+System manifests are identified via being named `gotk-components.y(a)ml` and containing comment headers as generated by `flux bootstrap`.
 
 Updating system manifests requires that either:
 
 1. The `flux` tool is pre-installed, or
 1. You run a Docker image based on [containerbase](https://github.com/containerbase), such as the official Renovate images, and have `binarySource=install` configured
 
```

Comment:
> I agree, it was leaning a little too much into implementation details. I've applied the suggested changes and removed the latter line about components.
> 
> On that thought, is it worth popping a warning about [custom flags not being supported](https://github.com/renovatebot/renovate/issues/13952#issuecomment-1031620639) for things like `--cluster-name`? In such a case Renovate will overwrite these values with the defaults in its PR.

Overwriting a users custom change is something that we should probably warn about. Here's the _code_ for a Material for MkDocs admonition. The `<!-- prettier-ignore -->` is needed to keep Prettier away. By default Prettier removes the 4 whitespaces we need for the admonition block. 😄 

```markdown
<!-- prettier-ignore -->
!!! warning
    Four blank spaces so the text lines up with the warning word.
    Add four blank spaces if you need another sentence.
```

I'll let you put the admonition warning in a logical place. 😉

Reply:
Ok, sounds good to me. I've added a warning and linked the issue.